src/webui/api/appcontroller.cpp

```diff
@@ -45,6 +46,7 @@
 #include <QTimer>
 #include <QTranslator>
 
+#include "apierror.h"
```

Comment:
wrong order. put it just above `"../webapplication.h"`

Reply:
Done